source/localizable/seminar/index.nl.html.erb

```diff
@@ -1 +1,247 @@
-Seminar here ...
```

Comment:
Doesn't 'seminar.nl.html.erb' do the trick, since we have directory indexes working here?